projects/solid-data-wallet.md

```diff
@@ -0,0 +1,80 @@
+
+The project is currently open source and hosted on [github](https://github.com/inrupt/inrupt-data-wallet) by Inrupt. 
+
+
+# Preferred Maturity Level
+
+The project is based on production ready technologies with significant adoption. Due to the wallet project being in early stage though, we would suggest it being adopted by [Labs](https://tac.openwallet.foundation/governance/project-lifecycle/#labs).
+
+# Project Description
+
+The proposed project is a [React Native](https://reactnative.dev/) and [Expo](https://expo.dev/) based mobile data wallet implementation.
+
+The project was initially developed by [Inrupt](https://www.inrupt.com/solutions/data-wallet), part of an effort to build a [Solid](https://solidproject.org/) backed wallet infrastructure so that *"organizations deliver innovative apps, products, and experiences \[...] powered by flexible, secure, and user-centric data sharing"*.
+
+The timing is also important, as Solid moved to the final stage of standardization at W3C, with the creation of the [Linked Web Storage](https://www.w3.org/groups/wg/lws/) workgroup. In addition to that, Inrupt decided to open source the wallet technology and make it available to a larger audience.
+
+# Alignment with the OpenWallet Foundation Mission
+[OpenWallet Foundation mission](https://tac.openwallet.foundation/governance/charter/) emphasizes three core elements:
+* *"develop and maintain open source code for wallets to enable and ensure wallet interoperability,"* - The project is very well aligned with this principle and, arguably, goes a step further in that it promotes a larger scoped data interoperability, via the Solid set of specifications. The wallet code proposed is, as expected, open sourced under the permissive ALv2 license.
+* *"advocate for the adoption of the interoperable digital wallet technology, \[...]"* - Inrupt, the company that originally developed the wallet is already strongly advocating for data interoperability. The Solid specs where initially imagined by Inrupt's CTO, Sir Tim Berners-Lee with the intent of extending *"the web to include identity management, access control and universal standards for data. These capabilities decouple data from applications so that data is organized around individuals."*
+* *"collaborate with Standards Development Organizations (SDOs) in the development and proliferation of open standards related to digital wallets"* - Inrupt will continue to be active in the definition of standards at W3C, as well as development and promotion of the wallet technologies backed by Solid and related specs, such as DID, verifiable credentials, and others.
+
+# Code of Conduct
+
+[OpenWallet Foundation code of conduct](https://tac.openwallet.foundation/governance/code-of-conduct/).
+
+# TAC Sponsor
+TBD
+
+# Project License
+[ALv2](https://www.apache.org/licenses/LICENSE-2.0) - SPDX:Apache-2.0
+
+# Source Control
+
+[OpenWallet Foundation Labs](https://github.com/openwallet-foundation-labs)
```

Comment:
```suggestion
The project is currently open source and hosted on [github](https://github.com/inrupt/inrupt-data-wallet) by Inrupt.
```

Reply:
My understanding is that the `Source Control` section refers to where we propose the project be hosted, which is OWF Labs. The change you suggested is actually a copy of line 11 that gives the current location of the project already.

The way I read the proposal template is that the `Source Control` section is about where we propose the project to be hosted after the OWF decides to adopt it. More specifically, should line 44 be deleted and replaced with a copy of line 11 or should line 44 stay (maybe with a clarification) and a copy of line 11 by added?

Reply:
From the template:
> If the project already exists, please provide a link to the repositories. If no source exists, this is okay. Please state that this will be a completely new project with code beginning in the OpenWallet Foundation.

The idea is that if the project already exists, we can take a look at the repos and examine code and other details on how the project is already functioning. I think that we can add where the project currently is and a note that the plan is to move these to OWF.